modules/develop/partials/run-transforms.adoc

```diff
@@ -20,7 +20,7 @@ Ensure that your Redpanda cluster has xref:manage:kubernetes/networking/external
 +
 TIP: For a tutorial on setting up a Redpanda cluster with external access, see xref:deploy:deployment-option/self-hosted/kubernetes/get-started-dev.adoc[].
 endif::[]
-- At least version 1.20 of https://go.dev/doc/install[Go^] installed on your host machine.
```

Comment:
@rockwotj Is TinyGo still supported? If so, I'll add a link to the [install instructions](https://tinygo.org/getting-started/install/).

Reply:
They don't need to install TinyGo, just standard Go: https://go.dev/doc/install

This line is needed for Golang development

If you want to do Rust development you need rustup installed (but not Golang).